index.html

```diff
@@ -72,6 +72,15 @@
       </div>
     </header>
 
+    <div class="alert" id="no-requirements-alert">
+      <span class="closebtn" onclick="this.parentElement.style.display='none';"
+        >&times;</span
```

Comment:
`&times;` is the multiplication sign for the 'x' button. I like it but for the sake of standardization, should I convert it fontAwesome?

Reply:
I think this is okay.